postgres/pgdump.go

```diff
@@ -620,6 +668,20 @@ func toSchemaKeys(conv *internal.Conv, table string, s []string) (l []schema.Key
 	return l
 }
 
+// toIndexKeys converts a string list of PostgreSQL primary keys to
```

Comment:
Replace "string list" with just list?
Since this isn't a string list.

Reply:
updated